### PR TITLE
Add vendor configuration support, example and README for Médiametrie analytics

### PR DESCRIPTION
**Pull request by loic-seguin**
Hi,

This PR is about to add support for Médiamétrie analytics.

This PR is associated to this issue #2189.

Thanks.

### CODE REVIEW

File: extensions/amp-analytics/0.1/vendors.js [hunk 1]
```diff
@@ -139,6 +139,33 @@ export const ANALYTICS_CONFIG = {
     }
   },
 
+  'mediametrie': {
+    'requests': {
+      'host': '//prof.estat.com/m/web',
```
Comment (cramforce) [inline extensions/amp-analytics/0.1/vendors.js]: Please use explicit https

File: extensions/amp-analytics/0.1/vendors.js [hunk 2]
```diff
@@ -139,6 +139,33 @@ export const ANALYTICS_CONFIG = {
...
+      'pageview': '${host}/${serial}?' +
+      'c=${level1}' +
```
Comment (rudygalfi) [inline extensions/amp-analytics/0.1/vendors.js]: Please indent by two additional spaces all of the additional lines corresponding to the `'pageview'` definition.

File: extensions/amp-analytics/amp-analytics.md [hunk 3]
```diff
@@ -113,6 +113,12 @@ Type attribute value: `atinternet`
 
 Adds support for AT Internet. More details for adding AT Internet support can be found at [developers.atinternet-solutions.com](http://developers.atinternet-solutions.com/javascript-en/advanced-features-javascript-en/accelerated-mobile-pages-amp-javascript-en/).
 
+##### Médiamétrie
```
Comment (rudygalfi) [inline extensions/amp-analytics/amp-analytics.md]: Please alphabetize the Médiamétrie entry with respect to the other providers

**Comment by loic-seguin** [comment]
We have already signed the CLA as a corporation named Médiamétrie. The email I used for the commit is loicseguin.webmaster@gmail.com. And this email belongs to the group who has signed the CLA which is equipe-mesure@googlegroups.com. Let us know if we did something wrong.

**Comment by loic-seguin** [comment]
I signed it!

**Comment by rudygalfi** [comment]
LGTM. cc @avimehta to take a look

**Comment by rudygalfi** [comment]
@loic-seguin can you squash your commits?

**Comment by avimehta** [comment]
lgtm.

**Comment by loic-seguin** [comment]
I just squash my commits into one 91caf82.

**Comment by loic-seguin** [comment]
I just rebased my commits into one 91caf82 @rudygalfi @cramforce.

**Comment by loic-seguin** [comment]
Hi,
The status of the pull request is blocked, what does it mean ?

**Comment by rudygalfi** [comment]
@loic-seguin We are holding merges until Monday to keep stability. We'll merge at that time.

Can you squash the commits into a single commit? There are steps here: https://github.com/ginatrapani/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit

**Comment by loic-seguin** [comment]
I created a new pull request with only one commit for my changes #2322. @rudygalfi 

Thanks

**Pull request closed without merging**